docs/reference-guides/README.md

```diff
@@ -16,11 +16,12 @@
 -   [Metadata](/docs/reference-guides/block-api/block-metadata.md)
 -   [Variations](/docs/reference-guides/block-api/block-variations.md)
 
-## [Filter Reference](/docs/reference-guides/filters/README.md)
+## [Hooks Reference](/docs/reference-guides/filters/README.md)
 
--   [Block Filters](/docs/reference-guides/filters/block-filters.md)
--   [Editor Filters (Experimental)](/docs/reference-guides/filters/editor-filters.md)
--   [Parser Filters](/docs/reference-guides/filters/parser-filters.md)
+-   [Block Hooks](/docs/reference-guides/filters/block-filters.md)
+-   [Editor Hooks](/docs/reference-guides/filters/editor-filters.md)
+-   [i18n Hooks](/docs/reference-guides/filters/i18n-filters.md)
```

Comment:
i18n was missing here.

Reply:
What is the process to also change the URL for the page, and have a 301 redirect? It would be great if it could be initiated in context of this documentation change. I understand that this will delay the merge a but with additional work being done on the documentation, we probably should solve that part as well.

Reply:
I [posted a question in #meta team channel](https://wordpress.slack.com/archives/C02QB8GMM/p1657728772768939), referring this PR/Comment thread.

Reply:
Thank you so much @bph. It would be also great to document the process once we know how it works. The document to include the details would be https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/documentation/README.md.

Reply:
@gziolo I filed a [meta trac ticket](https://meta.trac.wordpress.org/ticket/6407)to handle the redirects via manifest.json.